### ISSUE TITLE
[Dev] Screenshot investigation

### ISSUE THREAD

**Issue by catrobson** [fabric8-ui/fabric8-ux#521]
Investigate how we can do screenshots into a pop-up modal for feedback implementation.

**Comment by dlabrecq** [comment]
Using a library, named html2canvas, I was able to capture a screenshot and display that via a modal. I was also able to convert the snaphost 'canvas' object to a png image. 

"The screenshot is based on the DOM and as such may not be 100% accurate to the real representation as it does not make an actual screenshot, but builds the screenshot based on the information available on the page."

I also looked at a library, named jsfpdf, which converts the png to a pdf, client-side. This was used with CloudForms to save and print the snapshot.

That said, it's not clear what we will do with the snapshot. I believe the idea is to create a work item and include the snapshot there. However, work items do not support images. Even the new markup component will not display an image URL.
